digdag-standards/src/test/resources/io/digdag/standards/operator/for_each/basic_expected.yml

```diff
@@ -1,6 +1,6 @@
-+for-a=1:
++for-0=a=0=1:
```

Comment:
Why is the first `0` needed? Isn't it enough with `+for-a=0=1`?

Also, how about using `@` instead of `=` for indices? Using `=` for indices seems confusable with the next `=` for values to me.